assets/css/course-editor.scss

```diff
@@ -92,3 +93,39 @@ $primary: #6FCFB2;
 		}
 	}
 }
+
+.sensei-pricing-promo {
+
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	gap: 1em;
+
+	&__upgrade-content-text {
+		color: #a5a5a5;
```

Comment:
I've actually switched it for the opacity, instead of the color: [Link](https://github.com/Automattic/sensei/pull/4810/files#diff-47520f7e1c5b86d12dd032cfd96e86236dcc5c418968a61cab2f6ebd89f3d8f2R100)